### PR TITLE
Adding pipenv support to Local Installation

### LINKED ISSUE #369: Adding pipenv support to the installation documentation
**I'm submitting a**
- [x] feature request

**Current behavior:**
`pipenv` is not supported yet.

**Expected behavior:**
A `pipenv` support by adding how one can use it to deploy this project in the `Local Installation`.

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by realslimshanky**
Fixes #369 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- `pipenv` support is added to Local Installation documentation. `pipenv` is python.org recommendation and is better than using `pip`
and `virtualenv` separately.

### CODE REVIEW

**Comment by realslimshanky** [comment]
I've fixed conflicts with my PR.

File: docs/installation/local.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ venv\Scripts\activate # Windows
...
 ```
 
+or to use [`pipenv`](https://docs.pipenv.org) instead of `pip` and `virtualenv` separately.
+
+```
+npm install -g bower
+bower install
+pip install pipenv
+pipenv --two # To setup python 2 virtual environment
+pipenv install -r requirement.txt
```
Comment (bhaveshAn) [inline docs/installation/local.md]: Its `requirements.txt`

Comment (realslimshanky) [reply docs/installation/local.md]: Updated!

**Comment by realslimshanky** [comment]
I've fixed my branch w.r.t master of this repository.

**Comment by realslimshanky** [comment]
Thanks @bhaveshAn for the review :)

**Pull request merged**